hudi-utilities/src/test/resources/streamer-config/source-timestamp-millis.avsc

```diff
@@ -0,0 +1,162 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+{
+  "type" : "record",
+  "name" : "triprec",
+  "fields" : [
+    {
+      "name": "timestamp",
+      "type": "long"
+    },
+    {
+      "name": "_row_key",
+      "type": "string"
+    },
+    {
+      "name": "partition_path",
+      "type": "string"
+    },
+    {
+      "name": "trip_type",
+      "type": {
+        "type": "enum",
+        "name": "TripType",
+        "symbols": [
+          "UNKNOWN",
+          "UBERX",
+          "BLACK"
+        ],
+        "default": "UNKNOWN"
+      }
+    },
+    {
+      "name": "rider",
+      "type": "string"
+    },
+    {
+      "name": "driver",
+      "type": "string"
+    },
+    {
+      "name": "begin_lat",
+      "type": "double"
+    },
+    {
+      "name": "begin_lon",
+      "type": "double"
+    },
+    {
+      "name": "end_lat",
+      "type": "double"
+    },
+    {
+      "name": "end_lon",
+      "type": "double"
+    },
+    {
+      "name": "distance_in_meters",
+      "type": "int"
+    },
+    {
+      "name": "seconds_since_epoch",
+      "type": "long"
+    },
+    {
+      "name": "weight",
+      "type": "float"
+    },
+    {
+      "name": "nation",
+      "type": "bytes"
+    },
+    {
+      "name": "current_date",
+      "type": {
+        "type": "int",
+        "logicalType": "date"
+      }
+    },
+    {
+      "name": "current_ts",
+      "type": {
+        "type": "long",
+        "logicalType": "timestamp-millis"
```

Comment:
Could we also add `timestamp-micros` (also supported in V1), and other timestamp type like `time-millis` and `local-time-micros` for testing?

Reply:
No, the reason this works is that we're using an existing type in the datagen. Otherwise we need a new schema. the other test covers all the types. I have a standalone for this one because that was the type that was asked for

Reply:
Sg